variables.js

```diff
@@ -0,0 +1,17 @@
+function variables() {
```

Comment:
What is the purpose of this function? As of now, this function will never run.

Reply:
it would just help me display the user input to the html page. i think it was for a mini task you gave me on call. dont really need it at this point

Reply:
If the function is unused, lets remove it.

Reply:
Which means that all my other comments about this function are irrelevant for now ig, but still you should take the time to read through them as the comments explain good general practice for coding in JS (javascript).